raptor/playback.py

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from __future__ import absolute_import
+
+import os
+
+from raptor.playback_tools.mitmproxy.mitmproxy import Mitmproxy
+
+from mozlog import get_proxy_logger
+
+here = os.path.abspath(os.path.dirname(__file__))
+playback_dir = os.path.join(os.path.dirname(here), 'playback')
+LOG = get_proxy_logger(component='playback')
+
+
+class RaptorPlayback(object):
```

Comment:
What's the difference between this class and the Mitmproxy class? If the goal is to provide a base class that specific tools implement, you can use the `abc` module to formalize this, e.g `__metaclass__ = ABCMeta` and have `Mitmproxy` subclass `RaptorPlayback`.

Reply:
Yes that's the idea, so that we have a generic raptor playback class to make it easier switching out to a different playback tool. Didn't know how to do that, thanks!